modules/ibi-install-lcao-cli.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-install-lcao-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao} from the 4.15 Operator catalog.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the {lcao}.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-lifecycle-agent
+  annotations:
+    workload.openshift.io/allowed: management
+----
+
+.. Create the `Namespace` CR:
++
+[source,terminal]
+----
+$ oc create -f lcao-namespace.yaml
```

Comment:
here a specific file name is used , but the yaml above is unnamed.

Reply:
Updated